compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -231,6 +231,7 @@
     ITERABLE_NOT_SUPPORTED_OPERATION("iterable.not.supported.operation"),
     ITERABLE_TOO_MANY_VARIABLES("iterable.too.many.variables"),
     ITERABLE_NOT_ENOUGH_VARIABLES("iterable.not.enough.variables"),
+    ITERABLE_REQUIRES_N_VARIABLES("iterable.requires.n.variables"),
```

Comment:
Can't we reuse one of `ITERABLE_NOT_ENOUGH_VARIABLES` or `ITERABLE_TOO_MANY_VARIABLES`?

Reply:
Ah this is actually not needed anymore.. Will get rid of this.